✨Tokens: Extend tokens representations

CosAnca opened the pull request:
- Add representation for box-shadow
- Add representation for border-width
- Add representation for border-style
- Add representation for border-radius
- Add representation for opacity
- Fix the search width issue on the documentation site

CosAnca commented:
This PR addresses issue #70 by extending the tokens representations for other CSS properties, like `box-shadow`, `border-width`, `border-style`, `border-radius` and `opacity`. It also includes a fix for the search bar width on the documentation site.

dennisreimann commented:
Yeah, very cool – I really like this 👍 

Can you add the new token types to [the docs](https://github.com/dennisreimann/uiengine/blob/master/docs/design-tokens.md) as well? That'd be awesome, I'll merge this right after this is done. Good job, again!

For those curious, here's a preview …

![tokens](https://user-images.githubusercontent.com/886/87246508-2bd39080-c44e-11ea-83af-7784d0e24e37.png)

CosAnca commented:
I have updated the tokens documentation to include newly added types. Also changed `shadow` token type to `box-shadow` to be more inline with the actual CSS property.
